src/core.js

```diff
@@ -425,6 +425,16 @@ jQuery.extend({
 	support: support
 });
 
+// jsHint would error on this code due to the Symbol not being defined in ES5.
+// Defining this global in .jshintrc would create a danger of using the global
+// unguarded in another place, it seems safer to just disable jsHint for these
+// three lines.
+/* jshint ignore: start */
+if ( typeof Symbol === "function" && Symbol.iterator ) {
```

Comment:
@markelog 

> What do you mean by that? Did i miss something? What shim?

http://blog.jquery.com/2014/10/29/jquery-3-0-the-next-generations/ <-- Is this still legit?


> for..of is by no means the only way to use iterators - Array.from($collection) will use it too, and that can be polyfilled (and is, in es6-shim)

Yes, but `Array.from` also works with any array-like that doesn't have Symbol.iterator.